vue.config.js

```diff
@@ -40,6 +41,16 @@ module.exports = {
 					Object.assign( {}, options, { filename: `${filePrefix}[name].css` } ),
 					...args,
 				] );
+
+			config.module
+				.rule('images')
+				.test( /\.(png|jpe?g|gif|svg)(\?.*)?$/ )
+				.use( 'url-loader' )
+				.loader( 'url-loader' )
+				.options( {
+					limit: -1,
+					name: assetsPath + '/[name].[ext]',
```

Comment:
If we do not include `[path]` here somehow, this will only work if all assets are directly in `/assets`, without sub-directories. Was that an active decision?

Reply:
Was a "good enough" decision (no sub folders ATM) but would definitely benefit from future-proofing this.